Practice basic JQuery/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" type="text/css" href="styles.css">
```

Comment:
The `styles.css` seems to be missing, I get error when running the files. Please address this, Thanks

<img width="519" alt="image" src="https://github.com/user-attachments/assets/d84cd29e-a67d-4ede-b060-c85a0b420c89">